main.go

```diff
@@ -28,27 +34,34 @@ func main() {
 		os.Exit(1)
 	}
 
+	location := endpoints.AwsPartition().Regions()[*regionFlag].Description()
+
+	pricing, err := getDynamoPricing(location)
```

Comment:
This error doesn't seem to be checked. It would be normal to do that, e.g. if err != nil, then print an error and quit.